### ISSUE TITLE
Cannot get past 2FA authentication

### ISSUE THREAD

**Issue by maaezis** [merdok/homebridge-miot#409]
Hi, I'm new to homebridge and miot is actually my first plugin, so I might be doing something wrong, but I just can't get past the 2FA authentication of xiaomi account when trying to discover all my devices. I have my homebridge installed on Synology NAS. When I try to login, I get the 2FA prompt, I authenticate myself, get the webpage with "ok" and when I click on "discover all devices"  again, nothing happens.
When searching Google, I found only one way to result this and that was to deactivate the 2FA. However, the option is not available in Xiaomi account anymore. Someone else suggested to create new account without a phone number and skipping a prompt for 2FA, but no prompt popped up for me. With the new account, I still get the 2FA prompt in miot and after authenticating, nothing happens.
Am I missing some simple solution or doing something wrong?

**Comment by merdok** [comment]
It is very strange that 2FA cannot be deactivated, but i know that a lot of accounts have it somehow not active, including mine. 
I know that there were some issue with 2FA, the problem i guess is that you need to do the 2FA verification in the same window where you login, if you do it in another tab or window then i guess it will not work.

**Comment by maaezis** [comment]
Okay, so I had to wait 24 hours, because I got locked out for more than 5 authentication tries. When I click on the authenticate button, new tab automatically opens. I tried what you suggested and instead of clicking on authenticate, I copied the link and opened it in the same tab, authenticated myself and then typed the IP address of my homebridge into the same tab to go back to the plugin. I also tried using the back button to go back to homebridge after authenticating, but neither option works. When I try to login to my Xiaomi account again, I still get the same message to authenticate.
Any other suggestions?
